app/models/user.rb

```diff
@@ -8,5 +8,50 @@ class User < ApplicationRecord
 
   has_many :posts
   has_many :comments, dependent: :destroy
-  has_many :likes, dependent: :destroy
+  has_many :post_likes, dependent: :destroy
+  has_many :comment_likes, dependent: :destroy
+
+  has_many :senders, foreign_key: 'sender_id', class_name: 'Friendship'
+  has_many :receivers, foreign_key: 'receiver_id', class_name: 'Friendship'
```

Comment:
* [ ] User model `has_many :friendships` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friend.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))
* [ ] User model `has_many :inverted_friendships, class_name: "Friendship", foreign_key: "friend_id"` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friend.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))

- Even though you have the right associations the names should be changed so they make more sense with the project specs